models/comments.models.js

```diff
@@ -0,0 +1,15 @@
+const db = require("../db/connection")
+
+exports.selectCommentsById = (article_id) => {
+    const sqlString = `SELECT * FROM comments WHERE article_id = $1
+    ORDER BY comment_id DESC;`
+
+    return db.query(sqlString, [article_id])
+    .then((result)=>{
+        if(result.rows.length === 0){
+            return Promise.reject({status: 404, msg: 'article does not exist'})
```

Comment:
Is it an error if there are no comments? What about a brand new article? How many comments would that have?